### PR TITLE
Well log viewer

### PR DESCRIPTION
**Pull request by HansKallekleiv**

### CODE REVIEW

File: webviz_examples/webviz-full-demo.yml [hunk 1]
```diff
@@ -386,3 +396,13 @@ pages:
...
+
+  - title: Well log viewer
+    content:
+      - WellLogViewer:
+          wellfolder: ../observed_data/drogon_wells
+          wellsuffix: .rmswell
+          well_tvdmin: 1500
+          mdlog: MDepth
+          logtemplates:
+            - all_logs.yml
```
Comment (tnatt) [inline webviz_examples/webviz-full-demo.yml:408]: Duplicate?

**Review by tnatt — CHANGES_REQUESTED** [review]
Seems like you have added the well log viewer twice to the demo.
Otherwise lgtm 👍

File: webviz_examples/webviz-full-demo.yml (hunk 1 above)
Comment (HansKallekleiv) [reply webviz_examples/webviz-full-demo.yml:408]: well spotted :)

**Review by tnatt — APPROVED** [review]
👍

**Pull request merged**